src/api/spec/jobs/project_create_auto_cleanup_requests_job_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe ProjectCreateAutoCleanupRequests, type: :job, vcr: true do
+  include ActiveJob::TestHelper
+
+  describe '#perform' do
+    let(:admin) { create(:admin_user, login: 'Admin') }
+    let(:project) { create(:project, name: 'ProjectA') }
+    let(:attribute) { create(:auto_cleanup_attrib, project: project) }
+
+    subject { ProjectCreateAutoCleanupRequests.new.perform }
+
+    before do
+      allow(::Configuration).to receive(:cleanup_after_days).and_return(3)
```

Comment:
so there was a timebomb built in. It worked 3 days - and now fails :)

https://github.com/openSUSE/open-build-service/issues/8826